release.py

```diff
@@ -515,6 +517,64 @@ def display_help_job_chain_for_source_calls(args):
           f'{releasepy_check_url}')
 
 
+def get_collections_for_package(package_name, version):
+    script_directory = os.path.dirname(os.path.abspath(sys.argv[0]))
+    helper_script = f'{script_directory}/jenkins-scripts/dsl/tools/get_collections_from_package_and_version.py'
+    collection_yaml = f'{script_directory}/jenkins-scripts/dsl/gz-collections.yaml'
+    cmd = [helper_script,
+           get_canonical_package_name(package_name),
+           version,
+           collection_yaml]
+    _out, _err = check_call(cmd, IGNORE_DRY_RUN)
+    if _err:
+        print(f"An error happened running get_collections_from_package_and_version: {_err}")
+        sys.exit(1)
+    collection_list = _out.decode().strip().split(' ')
+    return collection_list
+
+
+def get_vendor_github_repo(package_name):
+    canonical_name = get_canonical_package_name(package_name)
+    return f"gazebo-release/{canonical_name.replace('-', '_')}_vendor"
+
+
+def create_issue_in_repo(github_repo, title, body):
+    _out = b""
+    # For parsing the \n correctly we need to store the content in a temp
+    # file and pass it as --body-file
+    with NamedTemporaryFile("w", delete=True) as f:
+        f.write(body)
+        f.flush()
+        cmd = ['ghxx', 'issue', 'create',
+               '--repo', github_repo,
+               '--title', title,
+               '--body-file', f.name]
+        try:
+            _out, _err = check_call(cmd)
+        except Exception as e:
+            _err = str(e)
+        if DRY_RUN:
+            return f"http://github.com/{github_repo}/issues/#dry-run-no-number", ""
+    return _out.decode().replace('\n', ''), _err
+
+
+def create_issue_in_gz_vendor_repo(args, ros_distro):
+    gz_vendor_repo = get_vendor_github_repo(args.package)
+    title = f'Update version for {ros_distro} to the latest tag of {args.package}: {args.version}'
+    body = f'The {get_canonical_package_name(args.package)} repository tagged a new: {args.version} '\
+           'This repository needs to be updated accordingly for the branch {ros_distro}:\n'\
+           ' * Sync to the new version in CMakelists.txt \n'\
+           ' * Bump the patch version in package.xml \n'\
+           ' * Run the release process for this ROS package'
+    _out, _err = create_issue_in_repo(gz_vendor_repo, title, body)
+    if _err:
+        print('  !! An error happened running the "gh issue" cmd. Do not run this script again.\n'
+              '     Please create the issue manually. The error reported was:\n'
+              f'     {_err}')
+        sys.exit(1)
+    return _out
```

Comment:
Is it possible to create a pull request instead? All we have to do is run https://github.com/gazebo-tooling/gz_vendor/blob/main/create_gz_vendor_pkg/create_vendor_package.py and give it the up-to-date package.xml from the upstream Gazebo library.